font.js

```diff
@@ -1,20 +1,21 @@
 var WebFont = require("webfont");
 
 exports.locate = function(load) {
-  return load.name;
+  return decodeURI(load.name).match(/[^/]*$/)[0];
```

Comment:
If there's a slash in the font name it won't work anyway will it? Are there any fonts with a slash in them?